v3/src/components/graph/adornments/movable-value/movable-value.tsx

```diff
@@ -10,13 +10,14 @@ import {valueLabelString} from "../../utilities/graph-utils"
 
 import "./movable-value.scss"
 
-export const MovableValue = (props: {
+interface IProps {
   model: IMovableValueModel
   axis: INumericAxisModel
   transform: string
-}) => {
-  const {model, axis, transform} = props,
-    layout = useAxisLayoutContext(),
+}
+
+export function MovableValue ({model, axis, transform}: IProps) {
```

Comment:
This should presumably be an `observer` as well, but you can wait until you're actually working on the `MovableValue` to fix it.